.travis.yml

```diff
@@ -1,42 +1,31 @@
 sudo: true
 dist: trusty
-
 os:
-  - linux
-  - osx
-osx_image: xcode8.2
-
-compiler:
-  - gcc
-  - clang
-
+- linux
 language: node_js
-
 node_js:
-  - "7.9.0"
-
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-    packages:
-      - g++-4.8
-      - desktop-file-utils
-      - xdg-utils
-
-cache:
-  directories:
-    - node_modules
-
-before_install:
-  - "[[ ${TRAVIS_OS_NAME} = linux ]] && export CXX=g++-4.8 || echo 'skipped'"
-
+- '7'
 install:
-  - node --version
-  - npm --version
-  # also downloads the build for release version
-  - NODE_ENV=test npm install
-
+- node --version
+- npm --version
```

Comment:
For consistency across projects, let's use `yarn` instead of `npm`. (You'll need to add it to the setup and cache also)